modules/standard/FileSystem.chpl

```diff
@@ -1139,28 +1150,29 @@ proc locale.umask(mask: int): int {
 }
 
 
-/* iter walkdirs(path: string=".", topdown=true, depth=max(int), 
-                 hidden=false, followlinks=false, sort=false): string
-  
-   walkdirs() recursively walks a directory structure, yielding
-   directory names.  The strings that are generated will be rooted
-   from 'path'.
-
-     * path: the directory to start from
-     * topdown: indicates whether to yield the directories using a
-       preorder (vs. postorder) traversal
-     * depth: indicates the maximal depth of recursion to use
-     * hidden: indicates whether to enter hidden directories
-     * followlinks: indicates whether to follow symbolic links or not
-     * sort: indicates whether to consider subdirectories in sorted
-       order or not
-  
-   by default, walkdirs() will start in the current directory, process
-   directories in preorder; recursively traverse subdirectories; and
-   neither follow dotfile directories nor symbolic links.  It will not
-   sort the directories by default.
-*/
+/* Recursively walk a directory structure, yielding directory names.
+   May be invoked in serial or non-zippered parallel contexts.
+
+   :arg path: the directory from which to start the walk (defaults to ".")
+   :type path: string
+
+   :arg topdown: indicates whether to yield a directory before or after descending into its children (defaults to `true`)
+   :type topdown: bool
+
+   :arg depth: indicates the maximum recursion depth to use (defaults to `max(int)`)
+   :type depth: int
 
+   :arg hidden: indicates whether to descend into hidden directories (defaults to `false`)
+   :type hidden: bool
+
+   :arg followlinks: indicates whether to follow symbolic links (defaults to `false`)
+   :type followlinks: bool
+
+   :arg sort: indicates whether or not to consider subdirectories in sorted order (defaults to `false`).  Note that requesting sorting has no effect in parallel invocations.
```

Comment:
Note that you can wrap this line, either to line up with ':arg' or 'indicates', and the html generation will still be the same.